src/parser/shared/metrics/apl/conditions/spellAvailable.tsx

```diff
@@ -18,7 +24,8 @@ export default function spellAvailable(
         return state;
       }
     },
-    validate: (state, _event) => state === null || state.isAvailable,
+    validate: (state, _event) =>
+      state === null || state.isAvailable || (inverse && !state.isAvailable),
```

Comment:
shouldn't this be:

```suggestion
    validate: (state, _event) =>
      state === null || (!inverse && state.isAvailable) || (inverse && !state.isAvailable),
```

otherwise you match when the spell is available even when inverted

Reply:
Yes, good catch - fixed